hack/multiplatform-build.sh

```diff
@@ -39,5 +42,12 @@ for PLATFORM in ${PLATFORMS} ; do
   fi
 
   echo "Building for $PLATFORM platform"
-  ${GO_BUILD_ENV} ${GO_CMD} build -ldflags="${LD_FLAGS}" -o ${BUILD_DIR}/${BUILD_NAME}-${GOOS}-${GOARCH}${EXTENSION} $1
+  FULL_NAME=${BUILD_NAME}-${GOOS}-${GOARCH}
+  ${GO_BUILD_ENV} ${GO_CMD} build -ldflags="${LD_FLAGS}" -o ${BUILD_PATH}/${FULL_NAME}${EXTENSION} $1
+
+  mkdir -p ${BUILD_PATH}/tmp/${FULL_NAME}
+  cp ${ROOT_PATH}/LICENSE ${BUILD_PATH}/tmp/${FULL_NAME}
+  cp ${BUILD_PATH}/${FULL_NAME}${EXTENSION} ${BUILD_PATH}/tmp/${FULL_NAME}/${BUILD_NAME}${EXTENSION}
+  (cd ${BUILD_PATH}/tmp && tar -czf ${BUILD_PATH}/${FULL_NAME}.tar.gz ${FULL_NAME})
```

Comment:
wouldn't this create the tar inside `tmp`?

Also, is it supposed to be one tar per platform, or one tar for all platforms?

Reply:
> wouldn't this create the tar inside tmp?

No. After build we have:

```
artifacts
├── kubectl-kueue-darwin-amd64
├── kubectl-kueue-darwin-amd64.tar.gz
├── kubectl-kueue-darwin-arm64
├── kubectl-kueue-darwin-arm64.tar.gz
├── kubectl-kueue-linux-amd64
├── kubectl-kueue-linux-amd64.tar.gz
├── kubectl-kueue-linux-arm64
├── kubectl-kueue-linux-arm64.tar.gz
├── kueue-chart-v0.8.0-devel-96-gbcce4105.tgz
├── manifests-alpha-enabled.yaml
├── manifests-dev.yaml
├── manifests.yaml
├── prometheus.yaml
└── visibility-api.yaml
```

It's because we are building to `${BUILD_PATH}/${FULL_NAME}.tar.gz`.

Reply:
> Also, is it supposed to be one tar per platform, or one tar for all platforms?

I think that one archive is also possible. But separate archives will be more optimised (not necessary to download all binaries on installation).